lib/chef/policy_builder/policyfile.rb

```diff
@@ -191,6 +194,11 @@ def setup_run_context(specific_recipes = nil, run_context = nil)
         run_context
       end
 
+      # Preserve the fall back to loading an unencrypted data bag item if the item we're trying to load isn't actually a vault item.
+      def set_databag_fallback
+        node.default["chef-vault"]["databag_fallback"] = true
```

Comment:
```suggestion
        node.default["chef-vault"]["databag_fallback"] = kitchen?
```

Reply:
I think we don't want it defaulting to `true` unless in test-kitchen?

Reply:
yeah, Agreed